source/sass/blocks/blocks-common/welcome.scss

```
@@ -0,0 +1,6 @@
+.welcome {
+  background-image: url("../img/decor-welcom-block.svg"), url("../img/image-welcom-block.png");
```

Обычно так читается полегче:

```suggestion
  background-image:
    url("../img/decor-welcom-block.svg"),
    url("../img/image-welcom-block.png");
```